woudc_data_registry/epicentre/instrument.py

```diff
@@ -163,13 +174,15 @@ def add(ctx, station, dataset, name, model, serial, geometry):
               help='identifier')
 @click.option('-st', '--station', 'station', help='station ID')
 @click.option('-d', '--dataset', 'dataset', help='dataset')
+@click.option('-t', '--deployment', 'deployment', help='deployment ID')
```

Comment:
You might want to make users supply both a `--station` and a `--contributor` instead of a full deployment ID, and construct the deployment ID like you do elsewhere.

Reason is, users may not know how a deployment ID is structured. Up to you on whether a user can supply only one of `--station` and `--contributor`, and what happens if they do.